test/verify/check-ovirt

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2016 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+from testlib import *
+
+import os
+
+# The oVirt image has following content
+# DC: "Default"
+# Cluster: "Default"
+# Host: "host" running in the image with plaintext communication to engine
+# VM: "VM" Blank-based VM with network boot 
+#
+
+WAIT_SCRIPT = """
+set -ex
+for x in $(seq 1 60); do
+    if [ "$(curl -s -k -u admin@internal:engine https://%(addr)s/ovirt-engine/api | xmllint --xpath "string(/api/product_info/name)" -)" = "oVirt Engine" ]; then
```

Comment:
xmllint smells like a new test dependency. Could this be replaced with a simple grep?

If that's not practical, can you please add it to HACKING? We would also need to update the cockpit test container for that, so I'd like to avoid it.